README.md

```diff
@@ -1 +1,108 @@
-# GSoC-2025-OpenFoodFacts
\ No newline at end of file
+# GSoC 2025 OpenFoodFacts
+
+## Project description
+
+- **Contributor:** Estrella Paoli
+- **Project:** [Implementation of a new structured nutrition schema](https://summerofcode.withgoogle.com/programs/2025/projects/mzpDaWXd)
+- **Organisation:** [Open Food Facts](https://world.openfoodfacts.org/)
+- **Repository:** [Open Food Facts server](https://github.com/openfoodfacts/openfoodfacts-server)
+- **Mentors:** Louis Bastarache, Stéphane Gigandet, Charles Nepote
+
+
+The Open Food Facts database stores data on products all around the world. 
+Even though the database has a lot of data in Europe, it could be completed in North America 
+and more particularly in Canada. The original main goal of this project was to complete the import 
+of products data from the USDA'S Food Data Central database that was initiated since January 2025. 
+At the beginning of the GSoC, it was then determined that it would be better to wait 
+until the new nutrition schema was integrated in the Open Food Facts database.
+
+A new nutrition schema was designed to solve current problems and difficulties it caused. 
```

Comment:
Link to google docs + mention that is was the fruit of a consultation with the community with evolutions during the project.